tests/common/auth_util.go

```diff
@@ -24,72 +24,72 @@ import (
 )
 
 const (
-	rootUserName = "root"
-	rootRoleName = "root"
-	rootPassword = "rootPassword"
-	testUserName = "test-user"
-	testRoleName = "test-role"
-	testPassword = "pass"
+	RootUserName = "root"
```

Comment:
Don't understand reason for making this public. If you plan to use it in linearizability tests, please don't.
Fact that we also need to setup authorization in multiple tests doesn't mean that we need to reuse make configuration global. I would prefer to avoid unnecessary dependendencies. 

If you want to create similar roles in linearizability tests, it would be better to just copy it. Copying is not bad as it implies that don't things are not related.

Reply:
Okay. Makes sense if isolation between linearizability tests and integration/e2e tests is more important. 

I will backout those changes then.